src/utils/code.ts

```diff
@@ -184,10 +184,20 @@ const buildSingleBlock = ({
       ${buildBlock({ component: childComponent, components, forceBuildBlock })}
       </${componentName}>`
     } else if (componentName === 'Conditional') {
-      content += `{${returnConditionalValue(
-        propsNames,
-        childComponent,
-      )}? <>${buildSingleBlock({
+      content += `{${
+        returnConditionalValue(propsNames, childComponent) === true ||
+        returnConditionalValue(propsNames, childComponent) === false
```

Comment:
Extract the value in variable first. See if you can improve the code, its not readable